content/methods/decide/user-stories.md

```diff
@@ -40,7 +40,7 @@ method:
 ## Additional resources{#add-personas}
 
 - [Agile — 18F Guides]({{ "/agile/" | url}}){.usa-link .usa-link--external}
-- [Agile software development — 18F De-risking Guide]({{ "/derisking/federal-field-guide/basic-principles/#agile-software-development" | url}}){.usa-link .usa-link--external}
+- [Agile software development — 18F De-risking Guide]({{ '/derisking-government-tech/principles/#principle-1-modern-software-development-practices' | url}}){.usa-link .usa-link--external}
```

Comment:
@ameliaswong @peterrowland (Non-blocking) What would you think about changing the link text to `Agile software development — 18F De-risking Government Technology Guide` here on the user stories method card? (Here it is in the [PR build](https://federalist-8bc0b42c-5fd0-4ae8-9366-cd9c265a4446.sites.pages.cloud.gov/preview/18f/guides/drg-v2-update-guides-links/methods/decide/user-stories/).)

I think we should do it to keep the naming of the new guide consistent, but it would cause the link to break onto a second line so I can see the case for keeping it as-is too.

Reply:
Works for me @nateborr.